Overview.bs

```diff
@@ -1137,7 +1137,8 @@ The algorithm:
 1.  Check that the <var>patch map</var> has [=Format 2 Patch Map/format=] equal to 2 and is valid according to the requirements in
      [[#patch-map-format-2]] (requirements are marked with a "must"). If it is not return an error.
 
-2.  Initialize <var>last entry id</var> to 0, <var>current byte</var> to 0, and <var>current id string byte</var> to 0.
+2.  If the [=Format 2 Patch Map/entryIdStringData=] offset is 0 then initialize <var>last entry id</var> to 0. Otherwise initialize
```

Comment:
I haven't been tracking our terminology closely enough: How are we using "array" in this document generally? When I think about vectors, strings, and arrays, the former are variable length while the latter isn't, but I guess that's a C++-centric way of looking at things. (So to me an "empty byte array" seems like something that would only be appropriate in certain degenerate cases.)

Reply:
Looking at other text related to this field we call it a byte string, so changed the wording here to "empty byte string".